bin/ch-image.py.in

```diff
@@ -171,6 +171,13 @@ def main():
    sp.add_argument("dest_ref", metavar="DEST_REF", nargs="?",
                    help="destination image reference (default: IMAGE_REF)")
 
+   # reset
+   help="resets ch-image builder storage"
```

Comment:
Would be good to use different words than the command to describe what it does. Also, this will release the bytes consumed, so good to allude to that.

```suggestion
   help="delete everything in builder storage"
```